class/devikyn; Variant Sorcerer.json

```diff
@@ -0,0 +1,3551 @@
+{
+	"$schema": "../schema.json", 
+	"_meta": {
+		"sources": [
+			{
+				"json": "VariantSorcerer_By_devikyn", 
+				"abbreviation": "VS", 
+				"full": "Variant Sorcerer", 
+				"authors": [
+					"/u/devikyn"
+				], 
+				"convertedBy": [
+					"MrVauxs"
+				], 
+				"url": "https://drive.google.com/file/d/15xiad-Kj0NifJ0y_YydN7ZbjZUXyibiE/view", 
+				"version": "1.0.0", 
+				"color": "944238"
+			}
+		], 
+		"dateAdded": 1594395239, 
+		"dateLastModified": 1594859805
+	}, 
+	"class": [
+		{
+			"name": "Variant Sorcerer", 
+			"source": "VariantSorcerer_By_devikyn", 
+			"hd": {
+				"number": 1, 
+				"faces": 6
+			}, 
+			"proficiency": [
+				"con", 
+				"cha"
+			], 
+			"spellcastingAbility": "cha", 
+			"startingProficiencies": {
+				"weapons": [
+					"daggers", 
+					"darts", 
+					"slings", 
+					"quarterstaffs", 
+					"light crossbows"
+				], 
+				"skills": [
+					{
+						"choose": {
+							"from": [
+								"arcana", 
+								"deception", 
+								"insight", 
+								"intimidation", 
+								"persuasion", 
+								"religion"
+							], 
+							"count": 2
+						}
+					}
+				]
+			}, 
+			"startingEquipment": {
+				"additionalFromBackground": true, 
+				"default": [
+					"(a) a {@item light crossbow|phb} and {@item crossbow bolts (20)|phb|20 bolts} or (b) any {@filter simple weapon|items|source=phb|category=basic|type=simple weapon}", 
+					"(a) a {@item component pouch|phb} or (b) an {@item arcane focus|phb}", 
+					"(a) a {@item dungeoneer's pack|phb} or (b) an {@item explorer's pack|phb}", 
+					"Two {@item dagger|phb|daggers}"
+				], 
+				"goldAlternative": "{@dice 4d4 × 10|4d4 × 10|Starting Gold}"
+			}, 
+			"multiclassing": {
+				"requirements": {
+					"cha": 13
+				}, 
+				"entries": [
+					{
+						"type": "entries", 
+						"name": "Spell Slots", 
+						"entries": [
+							"Any spell slots you gain from the Spellcasting feature are converted to sorcery points. Once you have added together your levels as prescribed in the Spell Slots section, take the resulting total and refer to the Sorcerer table for that level to determine the amount of sorcery points you have. For example, a character with 3 sorcerer levels and 2 paladin levels counts as a 4th-level sorcerer for the purposes of determining that character's total sorcery points."
+						]
+					}
+				]
+			}, 
+			"classTableGroups": [
+				{
+					"colLabels": [
+						"Metamagics Known"
+					], 
+					"rows": [
+						[
+							"—"
```

Comment:
Why not use `0`? Are you using a specific dash?

Reply:
That's how it is shown in the pdf, and hey, Monks Ki Points, Normal Sorcerers Sorcery Points, etc. also have such formatting.

Reply:
Yes, but `0` gets rendered as a dash too, and is better for automation